src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.csproj

```diff
@@ -111,9 +122,20 @@
     </ItemGroup>
     <PropertyGroup>
       <PathToMergedNuGetPackResource>$(ILMergeResultDir)NuGet.Build.Tasks.Pack.resources.dll</PathToMergedNuGetPackResource>
-      <IlmergeCommand>$(ILMergeExePath) /out:$(PathToMergedNuGetPackResource) $(ILMergePlatformArg) /lib:$(OutputPath)$(RuntimeIdentifier)\publish /internalize /xmldocs /log:$(ILMergeResultDir)IlMergeLog.txt</IlmergeCommand>
-      <IlmergeCommand Condition="Exists($(MS_PFX_PATH))">$(IlmergeCommand) /delaysign /keyfile:$(MS_PFX_PATH)</IlmergeCommand>
-      <IlmergeCommand>$(IlmergeCommand) $(OutputPath)$(CurrentCulture)NuGet.Build.Tasks.Pack.resources.dll @(ResourceArtifacts, ' ')</IlmergeCommand>
+      <IlmergeCommand>$(ILMergeExePath) ^
+        /out:$(PathToMergedNuGetPackResource) ^
+        $(ILMergePlatformArg) ^
+        /log ^
+        /lib:$(OutputPath)\publish ^
+        /internalize ^
+        /xmldocs ^
+        /log:$(ILMergeResultDir)IlMergeLogResources.txt</IlmergeCommand>
+      <IlmergeCommand Condition="Exists($(MS_PFX_PATH))">$(IlmergeCommand) ^
+        /delaysign ^
+        /keyfile:$(MS_PFX_PATH)</IlmergeCommand>
+      <IlmergeCommand>$(IlmergeCommand) ^
+        $(OutputPath)$(CurrentCulture)NuGet.Build.Tasks.Pack.resources.dll ^
+        @(ResourceArtifacts, ' ^&#x0A;')</IlmergeCommand>
```

Comment:
Did ' ' cause problems? I'm, asking because this is unreadable.

Reply:
Ibid. https://github.com/NuGet/NuGet.Client/pull/3511#discussion_r454570101